src/Module/Contact.php

```diff
@@ -290,11 +290,17 @@ public static function content(array $parameters = [], $update = 0)
 				$contact_id = $data['user'];
 			}
 
-			$contact = DBA::selectFirst('contact', [], ['id' => $contact_id, 'uid' => [0, local_user()], 'deleted' => false]);
-
-			// Don't display contacts that are about to be deleted
-			if ($contact['network'] == Protocol::PHANTOM) {
-				$contact = false;
+			if (empty($data)) {
```

Comment:
`If (!empty(..)` does make more sense, since an empty `$data` is an indicator invalid contact data - and reverse.